cypress/integration/manifest.js

```diff
@@ -5,6 +5,10 @@ const testManifest200s = service => {
   it(`should return a 200 status code for ${service}`, () => {
     testResponseCode(`/${service}/articles/manifest.json`, 200);
   });
+
+  it(`should return a 200 status code for ${service} without /articles`, () => {
```

Comment:
This test sadly wont work as on test and live (in production we go through mozart which arnt routed yet)

| env's | path |
|:------|:-----|
| all   | /news/articles/manifest.json |
| local | /persian/articles/manifest.json |
| local | /igbo/manifest.json |
| local | /yoruba/manifest.json |
| local | /yoruba/manifest.json |

You can limit a test to local only with the `describeForLocalOnly` describe function